ultravox/model/ultravox_model.py

```diff
@@ -55,6 +55,24 @@ def __init__(self, config: UltravoxConfig):
             self.audio_tower._no_split_modules or []
         )
 
+        audio_latency_bsize = config.audio_latency_bsize
+        audio_streaming_mask = None
+        if audio_latency_bsize != -1:
+            audio_latency_nblocks = (self.audio_tower.config.max_source_positions
+                * self.audio_tower.conv1.stride[0]
+                * self.audio_tower.conv2.stride[0]
+            )
+            audio_latency_nblocks //=  2 * audio_latency_bsize
```

Comment:
Actually, if `max_source_positions`  is not divisible by `audio_latency_bsize`, we might be in trouble, so I'd rather put an `assert` here.

Reply:
I don't think this was answered. The `*2` is still there and I don't see an explanation here or in the code about it. It would be ideal to avoid magic numbers as much as possible.